vendor/github.com/emicklei/go-restful/v3/log/log.go

```diff
@@ -0,0 +1,34 @@
+package log
```

Comment:
why is this file included?

Reply:
the previous commit this file got left out. (tests passed as i added a "go mod tidy/vendor" explicitly. somehow (gitignore?) this file got left out and i had to do a "git add" by "force" to do this in this PR